src/python/WMCore/MicroService/MSPileup/MSPileupTaskManager.py

```diff
@@ -38,11 +38,12 @@ def __init__(self, msConfig, **kwargs):
         self.cleanupDaysThreshold = msConfig.get('cleanupDaysThreshold', 15)
         dryRun = msConfig.get('dryRun', False)
         rucioScope = msConfig.get('rucioScope', 'cms')
+        customRucioScope = msConfig.get('customRucioScope', 'group.wmgroup')
```

Comment:
Unless I miss something this should also be group.wmcore

Reply:
And I confirm that group.wmcore is already in Rucio

Reply:
Eric, thanks for prompt review and good catch. I'm still was awaiting jenkins to see how last commits are played. But I changed the custom rucio scope value to use `group.wmcore` value.